src/runner/test-run-controller.js

```diff
@@ -127,10 +126,18 @@ export default class TestRunController extends EventEmitter {
 
         testRun.start();
 
-        const pageUrl           = testRun.test.pageUrl;
-        const needBypassHost    = this.opts.proxyBypass && checkUrl(pageUrl, this.opts.proxyBypass.split(','));
-        const externalProxyHost = needBypassHost ? null : this.opts.externalProxyHost;
+        const pageUrl             = testRun.test.pageUrl;
```

Comment:
Now, we use the self-writing module [match-url-wildcard](https://www.npmjs.com/package/match-url-wildcard) for match urls that should be ignored.
I propose to the another module for this - [minimatch](https://www.npmjs.com/package/minimatch).
Pros:
- well-known  glob syntax
- very popular - 42 million per month, a lot of tests

What do you think about it?
@AlexanderMoskovkin @AndreyBelym @AlexKamaev

Reply:
One the one hand `minimatch` syntax is well-known, on the other `match-url-wildcard` is tiny and contains some url-specific features. Does `minimatch` supports array of rules?
Could you write how some of url-specific rules from `match-url-wildcard` tests will look with the `minimatch` syntax?

@DevExpress/testcafe-docs what do you guys think?

Reply:
I know neither `minimatch` nor `match-url-wildcard` syntax but judging by `npm` pages I better understand `match-url-wildcard` because it's designed specially for URLs and all the examples show URLs.

With `minimatch`,  I have questions like does it handle schemas the way `match-url-wildcard` does?

```
var rule = 'google.com';

expect(matchUrl('http://google.com', rule)).to.be.true;
expect(matchUrl('https://google.com', rule)).to.be.true;
```

Reply:
Ok. We will change own `match-url-wildcard` library at any time.